website/docs/r/windows_web_app.html.markdown

```diff
@@ -512,6 +512,20 @@ A `slow_request` block supports the following:
 
 * `path` - (Optional) The path for which this slow request rule applies.
```

Comment:
We should remove this from the docs as is it being deprecated
```suggestion
```

Reply:
I added the deprecation message right below this property, shall we still remove it from the docs?